### DISCUSSION TITLE
Megabyte consumption with Openmptcprouter

### DISCUSSION THREAD

**Discussion by Melania95** [Ysurac/openmptcprouter#3187] [General]
Hi all friends, I have a doubt about the consumption of megabytes when using a bonding of 2 internet connections with OMR. Specifically, for example, if I download a 100 megabyte file, in theory 50 megabytes would be deducted from one connection and the same from the other or am I wrong? Or perhaps if connection no. 1 is faster than the second it would absorb more megabytes (for example 75 megabytes for the first and 25 for the second) because it would complete the download sooner? Can you clarify my doubt? Thank you all, greetings.

**Comment by Ysurac** [comment]
Hi,
Connections are used based on latency, so higher priority to the connection with the lowest latency. So it's more the second case.

**Reply by Melania95** [reply]
Ok thank you so much...
